src/Composer/Package/PackageInterface.php

```diff
@@ -282,6 +283,7 @@ public function getSuggests();
      * directories for autoloading using the type specified.
      *
      * @return array Mapping of autoloading rules
+     * @psalm-return array<string, array<string, string>>
```

Comment:
So you mean something along ..

```suggestion
     * @psalm-return array{?psr-0: array<string, string>, ?psr-4: array<string, string>, ?classmap: list<string>, ?files: listr<string>}
```